EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,201 @@
+---
```

Comment:
```suggestion
---
eip: 7851
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

Reply:
updated in the same commit as https://github.com/ethereum/EIPs/pull/9193#discussion_r1898155717.

Reply:
got it. updated in commit [314ed2b](https://github.com/ethereum/EIPs/pull/9193/commits/314ed2b98598f5ad51c5c1a4277686ca232c1df9).